test/README.md

```diff
@@ -20,20 +21,22 @@ export STEWARD_TEST_TENANT=$(kubectl -n $STEWARD_TEST_CLIENT get tenants.steward
 echo $STEWARD_TEST_TENANT
 ```
 
-##Run framework tests to check if the test framework works correctly
+### Run framework tests to check if the test framework works correctly
```

Comment:
This should be part of Run tests IMHO

Reply:
I moved it out because it's not part the real tests